packages/foreman/foreman/foreman.spec

```diff
@@ -885,8 +897,11 @@ cp -r %%{%{name}_dir} ./%{_datadir} || echo 0 \\
 mkdir -p ./%{_localstatedir}/lib/%{name} \\
 cp -r %{_localstatedir}/lib/%{name}/db ./%{_localstatedir}/lib/%{name} || echo 0 \\
 unlink ./%{_datadir}/%{name}/db \\
-ln -sv \`pwd\`/%{_localstatedir}/lib/%{name}/db ./%{_datadir}/%{name}/db \\
 pushd ./%%{%{name}_dir} \\
+mkdir db/ \\
+cp -rf %{_datadir}/%{name}/db/* db/ \\
```

Comment:
Doesn't this work:
```suggestion
cp -rf %{_datadir}/%{name}/db ./ \\
```

Reply:
I don't know. But this may change depending on how much of this nulldb code we put into Foreman. I wouldn't focus too much on the code itself as much as the "process" that is required. I'll open a Foreman PR with some of the proposed idea changes to help.